src/lib/nodes/dash/DashNode.ts

```diff
@@ -31,4 +42,17 @@ export class DashNode extends Node {
       ping: Date.now() - time
     }
   }
+
+  private async fetchNodeVersion(): Promise<void> {
+    try {
+      const { data } = await this.client.post<FetchNodeVersionResponse>('/', {method:'getnetworkinfo'})
+      if (data?.result.buildversion) {
```

Comment:
Must be required, according to `FetchNodeVersionResponse`

```suggestion
      if (data.result.buildversion) {
```